README.md

```diff
@@ -151,6 +151,7 @@ These are the VS Code [settings] available for the Extension:
 | `dvc.dvcPath`                          | Path or shell command to the DVC binary. Required unless Microsoft's [Python extension] is installed and the `dvc` package found in its environment.     |
 | `dvc.pythonPath`                       | Path to the desired Python interpreter to use with DVC. Should only be utilized when using a virtual environment without Microsoft's [Python extension]. |
 | `dvc.experimentsTableHeadMaxHeight`    | Maximum height of experiment table head rows.                                                                                                            |
+| `dvc.studio.shareExperimentsLive`      | Automatically share all new experiment metrics and plots logged with DVCLive to Studio. This option will only take effect once Studio is connected.      |
```

Comment:
[Q] Can we get rid of this table? I don't see the point in it when the extension page provides this... but the extension page is broken:

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/37993418/222668769-b1cd9c4c-8390-4a3c-b3f4-05cbf8865267.png">

I'll raise a PR to get rid of `package.nls.json`.

Reply:
caused by https://github.com/microsoft/vsmarketplace/issues/489

Reply:
Fixed by https://github.com/iterative/vscode-dvc/pull/3390